### PR TITLE
Update about-plugins.md

### PR DESCRIPTION
**Pull request by felixrabe**
Fix regex equivalent example.

### CODE REVIEW

File: docs/introduction/about-plugins.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ For example, the code snippet above is equivalent to:
 
 ```js
 plugins: [
-    [ /mypath\/**.css/, CSSPlugin() ]
+    [ /mypath\/\*\*\.css/, CSSPlugin() ]
```
Comment (nchanged) [inline docs/introduction/about-plugins.md]: it's a valid RegExp I believe we should have `.*\.css`

Comment (felixrabe) [reply docs/introduction/about-plugins.md]: Nope, the equivalent Regexp of `"mypath/**.css"` is ` /mypath\/\*\*\.css/`.

Example (see also https://www.google.com/search?q=js+escape+string+to+regex and https://stackoverflow.com/a/6969486):

    $ node
    > function escapeRegExp(str) {
    ...   return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
    ... }
    undefined
    > new RegExp(escapeRegExp("mypath/**.css"))
    /mypath\/\*\*\.css/

Comment (felixrabe) [reply docs/introduction/about-plugins.md]: The characters '/' and '*' and '.' all need to be escaped.

Comment (Kompwu) [reply docs/introduction/about-plugins.md]: I agree w/ @felixrabe .

Comment (nchanged) [reply docs/introduction/about-plugins.md]: But ` /mypath\/\*\*\.css/` isn't a string. It's a REGEXP

**Comment by felixrabe** [comment]
After thinking about it for a while, I came to the conclusion that maybe my understanding of the example might not be correct. I'm closing this and opening a new issue just to report the invalid RegExp, so @nchanged can implement a correct fix instead of this (probably incorrect) PR.

**Pull request closed without merging**